src/Apps/Artist/Routes/AuctionResults/SingleAuctionResultPage/AuctionResultBackLink.tsx

```diff
@@ -0,0 +1,19 @@
+import { Box, ChevronIcon, Clickable, Flex, Text } from "@artsy/palette"
+import { useRouter } from "System/Router/useRouter"
+
+export const AuctionResultBackLink: React.FC = () => {
+  const { router } = useRouter()
+
+  return (
+    <Box py={[2, 1]}>
+      <Clickable onClick={() => router.go(-1)} minHeight={30}>
```

Comment:
If you land on this page from the SERP this doesn't do anything. This should be a link back to the artist page not a browser back.

Reply:
And specifying heights like this should be avoided always (though I think this should be replaced with the `TopContextBar` anyway)